### DISCUSSION TITLE
Help for a complete Beginner

### DISCUSSION THREAD

**Discussion by TimMII88** [ImageMagick/ImageMagick#5939] [Help] [answered]
### ImageMagick version

ImageMagick-6.4.2-Q8

### Operating system, version and so on

W10

### Description

Hey all,

So, For about 10 years I have been working for a Rubberstamp manufacturer and we use a program Called Page Assembler the take our png files and adds vector lines around them. Though now it is starting to age and not meeting all of our requirements. So here I am diving into the ImageMagick world. I do not know how to code, but I have been working with Low code no code software for around 4.5 years, so I have a logical mind set. I have an understanding of what I need to do, I just need some guidance on how to get started on this journey. Are there any Tutorials anywhere online? Anyways, If you have any questions go ahead and ask! I know Ill have plenty. Thanks for reading the rant!

**Comment by fmw42** [comment, marked as answer]
For novices, see

http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdownhttps://imagemagick.org/script/porting.php#cli
https://legacy.imagemagick.org/discourse-server/index.php
